src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.tsx

```diff
@@ -132,7 +137,12 @@ export const MostViewedFooter = ({
 				data-cy-ab-runnable-test={variantFromRunnable}
 			>
 				<section className={asideWidth}>
-					<h2 className={headingStyles}>Most popular</h2>
+					{!inDeeplyReadTestVariant && (
+						<h2 className={headingStyles}>Most popular</h2>
+					)}
+					{inDeeplyReadTestVariant && (
+						<h2 className={headingStyles}>Accross The Guardian</h2>
+					)}
```

Comment:
```suggestion
					{inDeeplyReadTestVariant ? (
						<h2 className={headingStyles}>Accross The Guardian</h2>
					) : (
						<h2 className={headingStyles}>Most popular</h2>
					)}
```

Reply:
This is perhaps a subjective thing but I think this ternary would be nicer here?

Reply:
👍